fineract-core/src/main/java/org/apache/fineract/commands/domain/CommandSource.java

```diff
@@ -136,6 +136,9 @@ public class CommandSource extends AbstractPersistableCustom<Long> {
     @Column(name = "loan_external_id", length = 100)
     private ExternalId loanExternalId;
 
+    @Column(name = "term_variation_id", length = 100)
+    private Long termVariationId;
```

Comment:
No need for new field. Using `resourceId` is more than enough for this use case.